packages/muelu/src/Utils/ClassList/SC-LO-GO-NO.classList

```diff
@@ -76,7 +76,7 @@ RebalanceBlockAcFactory
 RebalanceBlockInterpolationFactory
 RebalanceBlockRestrictionFactory
 RebalanceTransferFactory
-RegionRFactory
+RegionRFactory - #if defined(HAVE_MUELU_KOKKOS_REFACTOR)
```

Comment:
Do we need the Kokkos refactor branch for that or just Tpetra to be enabled?

Reply:
`HAVE_MUELU_TPETRA` seems to suffice.